src/runtime/internal/transitions.ts

```diff
@@ -27,7 +28,7 @@ interface Outro {
 
 function wait() {
 	if (!promise) {
-		promise = Promise.resolve();
+		promise = resolved_promise;
 		promise.then(() => {
```

Comment:
```suggestion
		resolved_promise.then(() => {
```

Reply:
For the `wait` function, we need to store the promise in a local variable. We can't use `resolved_promise` directly.